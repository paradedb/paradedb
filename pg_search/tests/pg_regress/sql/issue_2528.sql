SELECT id, paradedb.score(id) FROM mock_items_issue_2528 WHERE description @@@ 'shoes' and in_stock = true LIMIT 5;
