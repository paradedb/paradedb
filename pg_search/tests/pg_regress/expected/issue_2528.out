SELECT id, paradedb.score(id) FROM mock_items_issue_2528 WHERE description @@@ 'shoes' and in_stock = true LIMIT 5;
 id | score 
----+-------
  3 |      
  5 |      
(2 rows)

